nservicebus/sqlserver/configuration.md

```diff
@@ -46,6 +43,68 @@ By passing the transport a custom factory method which will provide connection s
 
 snippet:sqlserver-custom-connection-factory
 
+## Multiple connection strings
+
+In [*multi-catalog* and *multi-instance* modes](/nservicebus/sqlserver/deployment-options.md) it is necessary to include additional information in configuration.  The sender needs to know the connection string of the receiver, and the subscriber needs the connection string of the publisher.
+
+Connection strings for the remote endpoint can be configured in several ways:
+
+### Via the App.Config
+
+In Versions 2.1 to 2.x the endpoint-specific connection information is discovered by reading the connection strings from the configuration file with `NServiceBus/Transport/{name of the endpoint in the message mappings}` naming convention. If such a connection string is found, it is used for a given endpoint and this setting has precedence over the code-provided connection information.
+
+Starting from Version 3.0, the connection string for a remote endpoint has to be provided using configuration API.
+
+### Via the configuration API - Push mode
+
+In the push mode the whole collection of endpoint connection information objects is passed during configuration time.
+
+snippet:sqlserver-multidb-other-endpoint-connection-push
+
+
+### Via the configuration API - Pull mode
+
+The pull mode can be used when specific information is not available at configuration time. One can pass a `Func<String, ConnectionInfo>` that will be used by the SQL Server transport to resolve connection information at runtime.
+
+snippet: sqlserver-multidb-other-endpoint-connection-pull
+
+### Example
+
+Given the following mappings:
+
+snippet:sqlserver-multidb-messagemapping
+
+and the following connection strings:
+
+snippet:sqlserver-multidb-connectionstrings
+
+The messages sent to the endpoint called `billing` will be sent to the database catalog `Billing` on the server instance `DbServerB`. The messages to the endpoint called `sales` will be sent to the default database catalog and database server instance, because the endpoint specific configuration wasn't passed, i.e. `MyDefaultDB` on server `DbServerA`.
+
+## Custom database schemas
+
+The default schema in SQL Server transport is `dbo`.
+The schema for the specific endpoint can be configured using `DefaultSchema` method:
+
+snippet:sqlserver-non-standard-schema
+
+In Versions 1.2.3 to 2.x it was also possible to pass custom schema in the connection string, using `Queue Schema` parameter:
+ 
+snippet:sqlserver-non-standard-schema-connString
+snippet:sqlserver-non-standard-schema-connString-xml
+
+## Multiple custom schemas
+
+If two endpoints use different schemas then additional configuration is required. The sender needs to know the schema of the receiver, and subscriber needs the schema of the publisher. 
```

Comment:
How about something like:

If the endpoints are configured to use a different schema, then additional configuration is required for proper message routing. The sending endpoint needs to know the schema information of the receiving endpoint. The same applies when the publishing endpoint and subscribing endpoint use different schemas. The subscriber will need the schema information of the publisher in order to send subscription requests and to receive events from the publisher.